connectors/opa/lib/catalog_reader.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	clients "fybrik.io/fybrik/pkg/connectors/clients"
 	pb "fybrik.io/fybrik/pkg/connectors/protobuf"
+	datacatalogTaxonomyModels "fybrik.io/fybrik/pkg/taxonomy/model/datacatalog/base"
 	taxonomymodels "fybrik.io/fybrik/pkg/taxonomy/model/policymanager/base"
```

Comment:
rename to policyTaxonomyModels for consistency

Reply:
This entire file seems to be unused

Reply:
Removed catalog_reader and updated opa reader as many functions are not required now. Please review.